angular/core/components/thread_page/thread_page_controller.coffee

```diff
@@ -3,7 +3,7 @@ angular.module('loomioApp').controller 'ThreadPageController', ($scope, $routePa
 
   @requestedProposalKey = $routeParams.proposal or $location.search().proposal
   @requestedCommentId   = parseInt($routeParams.comment or $location.search().comment)
-  $location.url($location.path())
+  $location.url($location.path()) unless $location.search().position
```

Comment:
Can you explain the behaviour here, @HSalmon?
When will we see a search().position be present, what is the user doing in that case?